csi: enable pod networking by default for csi plugin pods (#10411)
rook/rook: Documentation/Troubleshooting/ceph-csi-common-issues.md

After the force delete, wait for a timeout of about 8-10 minutes. If the pod still not in the running state, continue with the next section to blocklist the node.

> 

Reviewer: How about a new section on how to get the IP of the holder pod?

Author: added a sub section to get the holder Pod IP.

### Blocklisting a node or holder Pod

To shorten the timeout, you can mark the node as "blocklisted" from the [Rook toolbox](ceph-toolbox.md) so Rook can safely failover the pod sooner.

If the Ceph version is at least Pacific(v16.2.x), run the following command:

> If csi host networking is enabled, we can blocklist the Node IP. If not, we need to blocklist the holderpod IP running on the faulty node.

Reviewer: How about "holder pod" as two separate words?
```suggestion
If csi host networking is enabled, we can blocklist the Node IP. If not, we need to blocklist the holder pod IP running on the faulty node.
```

After the force delete, wait for a timeout of about 8-10 minutes. If the pod still not in the running state, continue with the next section to blocklist the node.

### Blocklisting a node or holder Pod

To shorten the timeout, you can mark the node as "blocklisted" from the [Rook toolbox](ceph-toolbox.md) so Rook can safely failover the pod sooner.

If the Ceph version is at least Pacific(v16.2.x), run the following command:

If csi host networking is enabled, we can blocklist the Node IP. If not, we need to blocklist the holder pod IP running on the faulty node.

> #### Getting holder Pod IP

Reviewer: ```suggestion
### Holder Pod IP
```

```console

> kubectl get po -o wide -nrook-ceph

Reviewer: Is there a label we can use so they only get the holders? And a couple other minor suggestions on the formatting, I know it works either way, but suggesting this for readability.
```suggestion
kubectl get pod -o wide -n rook-ceph -l <label>
```
